pkg/controller/manager.go

```diff
@@ -8,18 +8,109 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/go-openapi/strfmt"
 	"github.com/google/uuid"
 
 	"github.com/cilium/cilium/api/v1/models"
 	"github.com/cilium/cilium/pkg/lock"
+	"github.com/cilium/cilium/pkg/option"
 )
 
 var (
 	// globalStatus is the global status of all controllers
 	globalStatus = NewManager()
 )
 
-type controllerMap map[string]*Controller
+type managedController struct {
```

Comment:
Nit: I think it's better to move unexported types to the bottom, after the exported ones, in order to improve readability for the package users.

Reply:
Fixed, thanks!